src/api/activiti-rest-api/api/processInstances.api.ts

```diff
@@ -599,7 +599,7 @@ export class ProcessInstancesApi extends BaseApi {
         let accepts = ['application/json'];
 
         let contextRoot = null;
-        let responseType = null;
+        let responseType = 'blob';
```

Comment:
should the swagger definition be updated? @eromano @dhrn

Reply:
i am getting response as :arrow_down: 

![image](https://user-images.githubusercontent.com/14145706/77788444-46445280-7087-11ea-9ce4-dca5bf85eb43.png)

Reply:
wierd, swagger is not listing this API

Reply:
maybe it's a manually created api, in that case it's ok just updating the response type

Reply:
ok